rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -255,6 +255,11 @@ namespace rsx
 		return static_cast<u32>((1ULL << 32) >> utils::cntlz32(x - 1, true));
 	}
 
+	static inline bool fcmp(float a, float b, float epsilon = 0.000001f)
```

Comment:
https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon

Reply:
and it really shouldnt be an rsx function only, some code in cellVdec can also benefit from this etc.

Reply:
Honestly real epsilon is too small for this. I have seen arithmetic drift propagate much faster. I guess we can have that for default though. I'll definately refactor into utils if it is needed there.